org/wildfly/ee/concurrent/context/service/capability.adoc

```diff
@@ -0,0 +1,24 @@
+NAME: org.wildfly.ee.concurrent.context.service
+
+DESCRIPTION: Provides a ContextService, as specified by Concurrency Utilities for Java EE (JSR 236).
+
+REGISTERED BY:
+
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: false
+
+SERVICE PROVIDED:
+
+  CLASS: org.jboss.as.ee.concurrent.ContextServiceImpl
```

Comment:
Can we not use javax.enterprise.concurrent.ContextService for the service type? Generally, we should avoid exposing specific implementation classes via the capability contract.

Reply:
is this really an issue with a "private" capability?

Reply:
If a capability is referenced by another subsystem, it is not "private".

Reply:
Additionally, a consumer of this capability should not need a dependency on the ee subsystem - just a dependency on the requisite API they want to use.

Reply:
it's not referenced by another subsystem

Reply:
it's not referenced by another subsystem

Reply:
OK - the linked jira indicated otherwise, hence my confusion.